src/golive/cdn/gac.md

```diff
@@ -0,0 +1,9 @@
+# Global Application Cache (Dual-CDN)
+
+A common configuration we recommend is to split CDN traffic between Fastly for dynamic content and CloudFront for static content, branded as our Global Application Cache.  Fastly offers far greater flexibility while CloudFront offers a better cost-per-megabyte.  However, this configuration requires application-level changes to ensure generated asset URLs point to the CloudFront CDN.  While Platform.sh provides recommendations for major applications on how to configure them accordingly, doing so is the customer's responsibility.
+
+The Global Application Cache is only available to Platform.sh Enterprise plans.
```

Comment:
```suggestion
The Global Application Cache is only available to Platform.sh Dedicated plans.
```

Reply:
Same as before.  Enterprise PLANS can include GAC, even on the grid.